app/src/main/res/layout/about_libraries_fragment.xml

```diff
@@ -3,23 +3,16 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
```

Comment:
Generally we put `fragment_` as a prefix, not suffix. Can you name the file `fragment_about_libraries`?